chronicle/Views/EnrollmentForm.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  EnrollmentForm.swift
+//  EnrollmentForm
+//
+//  Created by Giulia Campana on 8/19/21.
+//
+
+import SwiftUI
+
+struct EnrollmentForm: View {
+    
+    @ObservedObject var enrollmentViewModel = EnrollmentViewModel()
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading) {
+                AppHeader()
+                Text("Do you have an organization ID?")
+                    .padding(.bottom)
+                RadioButtonGroup(hasOrgId: $enrollmentViewModel.withOrgId)
+
+                if enrollmentViewModel.withOrgId == true {
+                    InputFieldView(label: "Enter your Organization ID", inputId: $enrollmentViewModel.organizationId, invalidInput: $enrollmentViewModel.invalidOrganizationId)
+                }
+                InputFieldView(label: "Enter your Study ID", inputId: $enrollmentViewModel.studyId, invalidInput: $enrollmentViewModel.invalidStudyId)
+                InputFieldView(label: "Enter your Participant ID", inputId: $enrollmentViewModel.participantId, invalidInput: $enrollmentViewModel.invalidParticipantId)
```

Comment:
Please use the same order across views. Either ...

1. participant id
2. study id
3. org id

... or ...

1. org id
2. study id
3. participant id

I prefer the first.

Reply:
oh ya, good point. thanks!